package.json

```diff
@@ -16,24 +16,8 @@
     "symphony-chat"
   ],
   "dependencies": {
-    "axios": "^0.21.1",
     "colors": "1.3.3",
-    "mem-fs": "^2.2.1",
-    "mem-fs-editor": "^9.3.0",
-    "mkdirp": "^1.0.4",
-    "node-forge": "^1.0.0",
-    "upath": "^2.0.1",
-    "yeoman-environment": "^3.6.0",
-    "yeoman-generator": "^5.4.2"
-  },
-  "devDependencies": {
-    "decamelize": "^5.0.0",
-    "fs-extra": "^10.0.0",
-    "jest": "^27.2.0",
-    "jest-junit": "^12.0.0",
-    "jest-mock-axios": "^4.4.0",
-    "yeoman-assert": "^3.1.1",
-    "yeoman-test": "^6.2.0"
+    "@finos/generator-symphony": "latest"
```

Comment:
I put latest here but we could also use 2.5.x

Reply:
For me it's ok to use `latest`